README.md

```diff
@@ -28,14 +28,14 @@ To use the code samples you need the following:
 
 You can checkout/run any chapter on its own. You can either [work directly on Github](https://help.github.com/) or
 [download the code as a ZIP file](https://github.com/kkapelon/java-testing-with-spock/archive/master.zip) For each chapter entering the command
-**mvn test** will compile and run all JUnit/Spock tests. 
+**mvn test** or **./gradlew test** will compile and run all JUnit/Spock tests. 
 
 ## IDE support
-Each chapter is self-contained. There is no super POM file. This repository is NOT a multimodule project.
-If you use Eclipse just import any chapter as a Maven project. If you use IntelliJ make sure that
-you import each chapter on its own project (and not as modules in a single project).
+Each chapter is self-contained. There is no super POM file. This repository is NOT a multi-module Maven or Gradle project.
+If you use Eclipse just import any chapter as a Maven project or run **./gradlew eclipse** and the Eclipse metadata files will be generated. You can optionally install the Gradle Buildship Eclipse plugin from the Eclipse Marketplace and import the build.gradle file from Eclipse. If you use IntelliJ make sure that
+you import each chapter on its own project (and not as modules in a single project). You can also run **./gradlew idea** from the command line and the IDEA metadata files will be automatically generated.
 To gain syntax highlighting you also need to install Groovy support in your IDE. This is optional
-because the build is fully automated via Maven.
```

Comment:
Hello

I would prefer that you leave the existing text unchanged. Then add a new paragraph called "Gradle support".
This will contain all the instructions you want. In the end it should "Gradle support was contributed by Mike Smithson and is completelly unofficial"

I would want readers to understand that Gradle support is something offered as a bonus. I certainly do not have time to support both build systems in the future